tests/unit/async_db/V2/test_cursor.py

```diff
@@ -841,3 +841,24 @@ async def test_cursor_iterate(
     with raises(CursorClosedError):
         async for res in cursor:
             pass
+
+
+async def test_server_side_header_database(
```

Comment:
Code reuse?

Reply:
Same as above. It would also be easier to remove V1 afterwards.